index.json

```diff
@@ -1,12 +1,7 @@
 [
   {
     "name": "check_run",
-    "actions": [
```

Comment:
Is this change normal and international?

Reply:
no, see my comment on #30. I guess the description was changed and the format in which the different actions are listed has changed

Reply:
see my comment at https://github.com/octokit/webhooks/pull/30#issuecomment-504851898